pages/ksphere/konvoy/latest/monitoring/recommendations/index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: layout.pug
+navigationTitle: Recommendations
+title: Recommendations
+menuWeight: 11
+excerpt: Recommended settings for monitoring and collecting metrics for Kubernetes, platform services, and applications deployed on the Konvoy cluster
+---
+
+
+This table provides recommended settings for certain addons, based on cluster size and expected capacity, in terms of pods, that satisfy the needs and workloads of our mixed workload testing (MWT)tests.
+
+<p class="message--note"><strong>NOTE: </strong>The resource settings reflect some settings but do not represent the exact structure to be used in the Konvoy `cluster.yaml`.</p>
```

Comment:
Pretty sure the `cluster.yaml` doesn't render on the site when inside `<strong>` tags. I think you have to wrap it in `<code>` html tags